src/engraving/libmscore/measure.cpp

```diff
@@ -234,7 +234,10 @@ void Measure::layoutStaffLines()
     for (MStaff* ms : m_mstaves) {
         if (isCutawayClef(staffIdx) && (score()->staff(staffIdx)->cutaway() || !visible(staffIdx))) {
             // draw short staff lines for a courtesy clef on a hidden measure
-            ms->lines()->layoutPartialWidth(width(), 4.0, true);
+            Segment* clefSeg = findSegmentR(SegmentType::Clef, ticks());
+            double partialWidth = clefSeg ? clefSeg->width() + clefSeg->minLeft() + score()->styleMM(Sid::clefLeftMargin)
+                                  + score()->styleMM(Sid::barWidth) : 0.0;
```

Comment:
This seems to assume that the barline is always a single barline. If that's not the case, the lines become too short:
<img width="502" alt="Scherm­afbeelding 2022-09-02 om 16 31 15" src="https://user-images.githubusercontent.com/48658420/188171857-4b77abcc-85d9-4cb3-92cd-d1544bac173a.png">
It would be ideal if we could use the actual width of the actual barline. Has that already been calculated at this point?

Reply:
Nah, that was just a stupid way of calculating it, I've now pushed a new one that should be correct independently of the barline. Good catch